src/main/java/core/basesyntax/strategy/impl/ReturnHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.HandlerAllOperation;
+import core.basesyntax.strategy.TransactionHandler;
+
+public class ReturnHandler implements HandlerAllOperation {
+    private final TransactionHandler processTransaction;
+
+    public ReturnHandler() {
+        processTransaction = new TransactionHandlerImpl();
```

Comment:
this object should be passed through the constructor

Reply:
not fixed